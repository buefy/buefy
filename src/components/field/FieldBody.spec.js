import { shallowMount } from '@vue/test-utils'
import BFieldBody from '@components/field/FieldBody'

describe('BFieldBody', () => {
    it('is called', () => {
        const wrapper = shallowMount(BFieldBody, {
            slots: {
                default: 'content'
            }
        })
        expect(wrapper.name()).toBe('BFieldBody')
        expect(wrapper.isVueInstance()).toBeTruthy()
    })
})
