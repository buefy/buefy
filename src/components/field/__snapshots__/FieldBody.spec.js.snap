// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BFieldBody render correctly 1`] = `<div class="field-body">content</div>`;
